### DISCUSSION TITLE
chrome os doesnt work

### DISCUSSION THREAD

**Discussion by ashyboiii** [3kh0/ext-remover#241] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
chrome os updated and now it doesnt work i think whats happening is every time i send a message with the code it resets the extensions that are built in by the administrator of the device

**Comment by SomePVZPlayerMatt** [comment]
it got patched sadly

**Comment by ILikeBypassing** [comment]
the chrome 106 update patched the update. the only thing WE can do rn is to wait for echo and whoever is working on this to fix it

**Reply by ehunt34** [reply]
Echo isn't working on it. He's just waiting for the TN Discord to come up with something so he can claim it as his own. We need to do something ourselves in order to change things.

**Reply by ILikeBypassing** [reply]
the dev team then

**Reply by ashyboiii** [reply]
i cant do anything accept suggest things so if you were going to do something you could try putting the bookmarklet as a start to a loop or  maybe code it into instead of chrome thinking its the webstore it thinks its somehow an admin or something

**Reply by ILikeBypassing** [reply]
looping the code wont work bc its not an admin patch, its a patch rolled to all chromebooks by the official chrome os team

**Comment by ashyboiii** [comment]
if we were

**Comment by ashyboiii** [comment]
for me the extension is enabled by an admin, for the extension im trying to disbable is gogaurdian it is installed by an administrator. which is the whole point of the code, to disable extensions set up by an admin or a person who set up the computer, so you could instead somehow send replace the message that sends telling the extension to enable on startup or somehow change the data on a profile making it to where an admin doesnt have acces to data on a computer by blocking it or something idk how code works but that is probaly impossible. but you could still send a message with the title admin instead of webstore

**Reply by ILikeBypassing** [reply]
you probably updated your chromebook without knowing just double check to make sure

**Comment by ashyboiii** [comment]
i honesly dont really know if this is possible

**Comment by ashyboiii** [comment]
it automatically updatees
